doc/release/upcoming_changes/23789.new_feature.rst

```diff
@@ -0,0 +1 @@
+Add copy parameter for api.reshape function.
```

Comment:
```suggestion
```

Reply:
The Pr to add more array API used this PR's number in its release note, causing a conflict. I will resolve it by removing the release note here, it is not really needed.

Reply:
Ooops I must have copied the wrong number. BTW have you considered just getting the number automatically instead of making people put it in the file name?

Reply:
I think this is a shortcoming of the towncrier tool we use. The number is used to create a link in the release docs. We could move to a different tool, [scriv](https://scriv.readthedocs.io/en/1.1.0/philosophy.html) comes to mind.